xql/src/xql/apply.py

```diff
@@ -85,79 +84,31 @@ def parse(a: t.Union[xr.DataArray, str], b: t.Union[xr.DataArray, str]) -> t.Tup
     return a, b
 
 
-def evaluate(a: t.Union[xr.DataArray, str], b: t.Union[xr.DataArray, str], operator: str) -> xr.DataArray:
+def apply_orderby(e: exp.Order, df: pd.DataFrame) -> pd.DataFrame:
     """
-    Evaluate the expression 'a operator b' using NumPy arrays.
+    Apply ORDER BY clause to the DataFrame.
 
-    Parameters:
-    - a (Union[xr.DataArray, str]): The first input value.
-    - b (Union[xr.DataArray, str]): The second input value.
-    - operator (str): The operator to be applied.
-
-    Returns:
-    - xr.DataArray: The result of the evaluation.
-    """
-    a, b = parse(a, b)
-    return operate[operator](a, b)
-
-
-def inorder(expression: exp.Expression, ds: xr.Dataset) -> xr.DataArray:
-    """
-    Evaluate an expression using an xarray Dataset and return the result.
-
-    Parameters:
-    - expression (exp.Expression): The expression to be evaluated.
-    - ds (xr.Dataset): The xarray Dataset used for evaluation.
+    Args:
+    - e (exp.Order): Parsed ORDER BY expression.
+    - df (pd.DataFrame): DataFrame to be sorted.
 
     Returns:
-    - xr.DataArray: The result of evaluating the expression on the given dataset.
+    - pd.DataFrame: Sorted DataFrame based on the ORDER BY clause.
     """
+    orderby_columns = []
+    orderby_columns_order = []
 
-    if(expression.key == "identifier"):
-        return ds[expression.args['this']]
-
-    if(expression.key == "literal"):
-        return expression.args['this']
-
-    args = expression.args
+    # Extract columns and sorting orders from the parsed ORDER BY expression
+    for el in e.expressions:
+        orderby_column = el.find(exp.Column).find(exp.Identifier).this
+        order = not bool(el.args['desc'])  # Descending if desc=False
```

Comment:
this will throw `KeyError` in case `desc` is not present.
i tested this out in a notebook

```
import sqlglot
query = "Select a, b from my_table order by a ASC, b";
expr = sqlglot.parse_one(query)
expr

output:
Select(
  expressions=[
    Column(
      this=Identifier(this=a, quoted=False)),
    Column(
      this=Identifier(this=b, quoted=False))],
  from=From(
    this=Table(
      this=Identifier(this=my_table, quoted=False))),
  order=Order(
    expressions=[
      Ordered(
        this=Column(
          this=Identifier(this=a, quoted=False)),
        desc=False,
        nulls_first=True),
      Ordered(
        this=Column(
          this=Identifier(this=b, quoted=False)),
        nulls_first=True)]))
```

as you can see in the second ordered column, desc key is not present.

Reply:
The ORDER BY keyword sorts the records in ascending order by default.
https://www.w3schools.com/sql/sql_orderby.asp

Reply:
Also 😅 ,

![image](https://github.com/google/weather-tools/assets/31622972/3ce1a4ca-38da-4054-bd67-bb8a17c30b11)

Reply:
checked, this works

Reply:
`el.args['desc']` will give `None` if `desc` is not provided.